keras/backend/tensorflow_backend.py

```diff
@@ -856,7 +856,7 @@ def update(x, new_x):
     """Update the value of `x` to `new_x`.
 
     # Arguments
-        x: A Variable.
+        x: A variable.
```

Comment:
I see. I wonder that it should attach `` ` `` symbols to Variable for understandability.
So I suggest to change to `Variable` in this case.